fix sanity test failures

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
Follow this guide: https://github.com/kubernetes-sigs/azuredisk-csi-driver/tree/master/test/sanity

Thanks for @Masquerade0097 's contribution, we have a lot more work to do to fix those sanity test failures:

 - below are the sanity test failure:
```
Summarizing 9 Failures:

[Fail] Node Service [It] should work 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/node.go:411

[Fail] Controller Service CreateVolume [It] should not fail when requesting to create a volume with already existing name and same capacity. 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:371

[Fail] Controller Service CreateVolume [It] should fail when requesting to create a volume with already existing name and different capacity. 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:442

[Fail] Controller Service DeleteVolume [It] should succeed when an invalid volume id is used 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:543

[Fail] Controller Service ValidateVolumeCapabilities [It] should fail when the requested volume does not exist 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:705

[Fail] Controller Service ControllerPublishVolume [It] should return appropriate values (no optional values added) 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:826

[Fail] Controller Service ControllerPublishVolume [It] should fail when the volume does not exist 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:883

[Fail] Controller Service ControllerPublishVolume [It] should fail when the node does not exist 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:941

[Fail] Controller Service ControllerUnpublishVolume [It] should return appropriate values (no optional values added) 
/home/priyanshu/work/src/github.com/csi-driver/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:1139

Ran 39 of 57 Specs in 133.694 seconds
FAIL! -- 30 Passed | 9 Failed | 0 Pending | 18 Skipped
--- FAIL: TestSanity (133.72s)
FAIL
FAIL	github.com/csi-driver/azuredisk-csi-driver/test/sanity	133.759s
Makefile:32: recipe for target 'test-sanity' failed
make: *** [test-sanity] Error 1
```

@Masquerade0097 you could start fix those failures, thanks for the work.

**What you expected to happen**:

**How to reproduce it**:

**Anything else we need to know?**:
There is some error like following which is due to config/setting issue, here `someNodeID` should be a VM name
```
failed to get azure instance id for node "someNodeID"
```

**Environment**:
- CSI Driver version: v0.2.0-alpha
- Kubernetes version (use `kubectl version`): 
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

there are still 7 failures now:
```
Summarizing 7 Failures:

[Fail] Node Service NodeGetInfo [It] should return approproate values
/root/go/src/github.com/kubernetes-sigs/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/node.go:146

[Fail] Node Service [It] should work
/root/go/src/github.com/kubernetes-sigs/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/node.go:385

[Fail] Controller Service CreateVolume [It] should not fail when requesting to create a volume with already existing name and same capacity.
/root/go/src/github.com/kubernetes-sigs/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:371

[Fail] Controller Service CreateVolume [It] should fail when requesting to create a volume with already existing name and different capacity.
/root/go/src/github.com/kubernetes-sigs/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:442

[Fail] Controller Service ControllerPublishVolume [It] should return appropriate values (no optional values added)
/root/go/src/github.com/kubernetes-sigs/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:802

[Fail] Controller Service ControllerUnpublishVolume [It] should return appropriate values (no optional values added)
/root/go/src/github.com/kubernetes-sigs/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go:1115

[Fail] Identity Service GetPluginInfo [It] should return appropriate information
/root/go/src/github.com/kubernetes-sigs/azuredisk-csi-driver/vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/identity.go:96

Ran 39 of 57 Specs in 124.390 seconds
FAIL! -- 32 Passed | 7 Failed | 0 Pending | 18 Skipped
--- FAIL: TestSanity (124.40s)

```

--- a/pkg/azuredisk/azuredisk.go
+++ b/pkg/azuredisk/azuredisk.go
@@ -36,6 +36,8 @@ import (
 )
 
 const (
+	driverName = "disk.csi.azure.com"
+
 	errDiskNotFound = "not found"
 	// default IOPS Caps & Throughput Cap (MBps) per https://docs.microsoft.com/en-us/azure/virtual-machines/linux/disks-ultra-ssd
 	defaultDiskIOPSReadWrite = 500

--- a/pkg/azuredisk/version.go
+++ b/pkg/azuredisk/version.go
@@ -29,7 +29,6 @@ var (
 	driverVersion = "N/A"
 	gitCommit     = "N/A"
 	buildDate     = "N/A"
-	driverName    = "N/A"
 	topologyKey   = "N/A"
 )
 

